gui/src/components/maintenance/cortx-auditlog.vue

```diff
@@ -168,16 +152,40 @@ export default class CortxAuditLog extends Vue {
     total_records: 1000
   };
 
+  @Watch("showDatePicker")
+  public watchClick(value: boolean) {
+    if (value) {
+      document.addEventListener('click', this.closeIfClickedOutside);
```

Comment:
1. We should avoid using document object directly.
2. We should not add event listeners to document root. We may encounter issues when event bubbles out.